src/busctl/busctl.c

```diff
@@ -459,6 +467,7 @@ static int find_nodes(sd_bus *bus, const char *service, const char *path, Set *p
                                "org.freedesktop.DBus.Introspectable", "Introspect",
                                &error, &reply, NULL);
         if (r < 0) {
+                notify_bus_error(&error);
```

Comment:
Maybe better to also pass `r`, and set `ERRNO` if `r < 0` but `error` is not set.

Reply:
we send ERRNO= anyway in the macro stuff that DEFINE_MAIN_FUNCTION() does.

(under the assumption the error is correctly propagated up, as it is here)

Reply:
Ah, indeed. Nice.